fluentd/plan.sh

```diff
@@ -0,0 +1,46 @@
+pkg_name=fluentd
+pkg_origin=core
+pkg_version=0.14.0
+pkg_deps=(
+  core/bundler
```

Comment:
I feel like bundler, gcc and make should be build dependencies. Does fluentd need them to run, or just to be built?

Reply:
Yes, you are correct. Thanks for pointing out.
Moved them to build dependencies and also removed `coreutils` as it's not needed.